### DISCUSSION TITLE
What replace ignoreNbf attribute in newer version ?

### DISCUSSION THREAD

**Discussion by Daniel-Ovejero** [panva/jose#627] [Q&A] [answered]
Hello we currently have trouble after migrating to a newer version. Indeed, the ignoreNbf property no longer exist in the VerifyOptions. What would be the replacement for that, or what should we do in order to skip that verification.

**Comment by panva** [comment, marked as answer]
The new versions do not offer to skip claims set timestamp validations.

If you want you can use compactVerify to just verify the JWS signature and then proceed with your own validation of the claims set.

**Reply by Daniel-Ovejero** [reply]
Thanks for your quick reply, I will try to make it work !
